bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/api1/Shelly1ApiJsonDTO.java

```diff
@@ -1193,6 +1194,15 @@ public static class ShellyExtSwitchStatusInput {
         // Shelly TRV
         public Boolean calibrated;
         public ArrayList<ShellyThermnostat> thermostats;
+
+        // Shelly BLU Remote
+        public Integer channel;
+        public String direction;
+        public Integer steps;
+        public Double rotationX;
+        public Double rotationY;
+        public Double rotationZ;
```

Comment:
These were changed from rotation1..rotation3 in the DTO in latest commit. So just checking: was it broken before, or is it broken now?